src/v/cluster/tx_gateway_frontend.h

```diff
@@ -25,6 +25,9 @@
 
 namespace cluster {
 
+using clock_type = ss::lowres_clock;
```

Comment:
FYI, we have `model::timeout_clock`, which is an alias to this, but helps describe the semantics.